managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -211,13 +211,8 @@ public void safeRun() {
         if (ml.hasActiveCursors()) {
             // Avoid caching entries if no cursor has been created
             EntryImpl entry = EntryImpl.create(ledger.getId(), entryId, data);
```

Comment:
Question: Does the `data` need to release?

Reply:
No need here. it will release at the end of this method or in `closeComplete`.

Reply:
Got it, thanks.